lib/cucumber_factory/factory.rb

```diff
@@ -4,12 +4,12 @@ class Error < StandardError; end
 
     ATTRIBUTES_PATTERN = '( with the .+?)?( (?:which|who|that) is .+?)?' # ... with the year 1979 which is science fiction
     TEXT_ATTRIBUTES_PATTERN = ' (?:with|and) these attributes:'
-    UPDATE_ATTR_PATTERN = '(?: (?:has|belongs to)( the .+?))?(?:(?: and| but|,)*( is .+?))?' # ... belongs to the collection "Fantasy" and is trending
+    UPDATE_ATTR_PATTERN = '(?: (?:has|belongs to)( the .+?)|(?: and| but|,)*( is .+?))' # ... belongs to the collection "Fantasy" | and is trending
     TEXT_UPDATE_ATTR_PATTERN = '(?: and|,)* has these attributes:'
 
     RECORD_PATTERN = 'there is an? (.+?)( \(.+?\))?' # Given there is a movie (comedy)
     NAMED_RECORD_PATTERN = '(?:"([^\"]*)"|\'([^\']*)\') is an? (.+?)( \(.+?\))?' # Given "LotR" is a movie
-    RECORD_UPDATE_PATTERN = 'the (.+?) (above|".+?"|\'.+?\')' # Given the movie "LotR" ...
+    RECORD_UPDATE_PATTERN = 'the ([a-zA-Z _-]+?) (above|".+?"|\'.+?\')' # Given the movie "LotR" ...
```

Comment:
It looks fine to me, but please do a double check on numbers.
If it's possible to have factories like `factory :my_user_2`, then it would now be excluded from this regex. If so, include `[0-9]`.

Or maybe we should go the other way and just exclude some characters: `[^'"<>:]` or similar.
This would still allow the user to name his models exotically, like with unicode chars :unicorn: